package.json

```diff
@@ -11,14 +11,18 @@
     "start": "node src/server.js",
     "dev": "src/node_modules/.bin/nodemon -L",
     "test": "echo \"No test specified\" && exit 0",
-    "deploy:clean": "docker image prune -f",
-    "deploy:build": "docker build . -t webiks/cron_server:latest",
-    "deploy:tag": "docker tag webiks/cron_server:latest webiks/cron_server:latest",
-    "deploy:push": "docker push webiks/cron_server:latest",
-    "upload": "npm run deploy:build && npm run deploy:tag && npm run deploy:push",
-    "deploy:remove": "docker rm -f webiks/cron_server",
-    "deploy:run": "docker run --name stress -p 3000:3000 -e webiks/cron_server",
-    "deploy:full": "npm run deploy:build && npm run deploy:remove && npm run deploy:run"
+    "docker:build": "sh -x ./scripts/docker-build.sh",
+    "docker:tag": "sh -x ./scripts/docker-tag.sh",
```

Comment:
No need for specific local tag as a separate operation.
Tagging version and latest should be done in a single operation.
Otherwise running local docker will run on an old version